app/services/otp_rate_limiter.rb

```diff
@@ -36,30 +31,21 @@ def lock_out_user
   end
 
   def increment
-    # DO NOT MEMOIZE
-    @entry = OtpRequestsTracker.atomic_increment(entry_for_current_phone.id)
     throttle.increment!
-    nil
   end
 
   def otp_last_sent_at
-    if IdentityConfig.store.redis_throttle_otp_rate_limiter_read_enabled
-      throttle.attempted_at
-    else
-      entry_for_current_phone.otp_last_sent_at
-    end
+    throttle.attempted_at
+  end
+
+  def throttle
+    @throttle ||= Throttle.new(throttle_type: :phone_otp, target: throttle_key)
   end
 
   private
 
   attr_reader :phone, :user, :phone_confirmed
 
-  # rubocop:disable Naming/MemoizedInstanceVariableName
-  def entry_for_current_phone
-    @entry ||= OtpRequestsTracker.find_or_create_with_phone_and_confirmed(phone, phone_confirmed)
```

Comment:
Same note about this method not being used anywhere and could be removed from `OtpRequestsTracker`.

Reply:
Just to clarify, are we planning to drop the `otp_requests_trackers` table altogether at some point?

Reply:
Yeah, plan is to drop everything related to OtpRequestsTracker in this, and drop the table if we're content with the functionality.